### PR TITLE
Footer update

### PR DESCRIPTION
**Pull request by Jibz-Mysten**
## Description 

Describe the changes or additions included in this PR.
Change the footer logo and copyright and social links 
Screenshot 2023-05-02 at 11 06 14 PM

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [x] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/explorer/src/components/footer/Legal.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export function LegalText() {
         <div className="flex justify-center md:justify-start">
             <Text color="steel-darker" variant="pSubtitleSmall/medium">
                 &copy;
-                {`${new Date().getFullYear()} Sui. All
+                {`${new Date().getFullYear()} Mystenlabs. All
```
Comment (williamrobertson13) [inline apps/explorer/src/components/footer/Legal.tsx:14]: I think this should be `Mysten Labs`

Comment (mamos-mysten) [inline apps/explorer/src/components/footer/Legal.tsx:14]: Mysten Labs

**Review by mamos-mysten — CHANGES_REQUESTED** [review]

**Review by williamrobertson13 — APPROVED** [review]
lgtm besides the copyright typo

File: apps/explorer/src/components/footer/Legal.tsx (hunk 1 above)
Comment (Jibz-Mysten) [reply apps/explorer/src/components/footer/Legal.tsx]: done

**Comment by mystie711** [comment]
please update this section
image

**Review by mamos-mysten — APPROVED** [review]

**Review by plam-ml — APPROVED** [review]

**Review by mystie711 — APPROVED** [review]

**Pull request merged**